README.md

```diff
@@ -242,6 +242,16 @@ In MVC views `<feature>` tags can be used to conditionally render content based
 </feature>
 ```
 
+The tag helper can encompass multiple features. By listing all feature names, separated by commas, and specifing the `requirement` in the example below, the content is only rendered if `FeatureX` and `FeatureY` are both enabled.

```

Comment:
```suggestion
The `<feature>` tag can reference multiple features by specifying a comma separated list of features in the name attribute.

/// HTML+Razor
<feature name="FeatureX,FeatureY">
  <p>This can only be seen if 'FeatureX' and 'FeatureY' are enabled.</p>
</feature>


By default, all listed features must be enabled for the feature tag to be rendered. This behavior can be overidden by adding the `requirement` attribute as seen in the example below.

/// HTML+Razor
<feature name="FeatureX,FeatureY" requirement="Any">
  <p>This can only be seen if either 'FeatureX' or 'FeatureY' or both are enabled.</p>
</feature>

```